CHANGELOG.md

```diff
@@ -45,6 +92,9 @@ This release requires a manual change in your galley configuration: `settings.co
 
 ## API Changes
 
+* A new team feature for classified domains is available (#1626):
+  - a public endpoint is at `GET /teams/:tid/features/classifiedDomains`
+  - an internal endpoint is at `GET /i/teams/:tid/features/classifiedDomains`
```

Comment:
Why did these things get added to previous release notes? Looks like a mistake.

Reply:
e7d682d you're right, thanks!  it was like this on `develop`, but i wasn't careful enough to catch it.